### PR TITLE
Fix 3051

### PR DESCRIPTION
**Pull request by antgonza**
Added the how-to for SCP and other common questions from the forum; it's just text so the review should be pretty quick.

### CODE REVIEW

File: qiita_pet/support_files/doc/source/faq.rst [hunk 1]
```diff
@@ -244,6 +244,89 @@ etc; for more information check
...
+To take advantage of this feature you need to:
+
+#. Prepare all the files you want to transfer in a single folder (or you can do this on
+   multiple folders). Note that Qiita will only collect files with valid extensions; see the
+   top of the "Upload Files" page within your study for the latests list of valid extensions.
+#. In your server (this needs to be run within your home directory in the server where you
+   store the files!), generate a new key by running:
+   `ssh-keygen -t rsa -C "ssh test key" -f ~/.ssh/qiita-key -P ""`. Here is where you are
+   creating that key to your storage-shed.
+#. Allow access using the new key to new connections (this also needs to be ran in the
```
Comment (adswafford) [inline qiita_pet/support_files/doc/source/faq.rst:270]: "needs to be ran" -> "needs to be run"

File: qiita_pet/support_files/doc/source/faq.rst [hunk 2]
```diff
@@ -244,6 +244,89 @@ etc; for more information check
...
+connecting to it via scp - in other words, this only works when moving files from a server to Qiita.
+
+Now, the way it works is that you need to create a new secure key to that server, imagine
+that you are making a copy of your storage-shed key, then you share that key to Qiita (you will give
+access to Qiita to run a single copy command in your server), Qiita uses that key and securely
+destroy it.
+
+To take advantage of this feature you need to:
+
+#. Prepare all the files you want to transfer in a single folder (or you can do this on
```
Comment (adswafford) [inline qiita_pet/support_files/doc/source/faq.rst:263]: can do this on multiple folders -> can transfer multiple times from multiple folders into one study

File: qiita_pet/support_files/doc/source/faq.rst [hunk 3]
```diff
@@ -244,6 +244,89 @@ etc; for more information check
...
+   multiple folders). Note that Qiita will only collect files with valid extensions; see the
```
Comment (adswafford) [inline qiita_pet/support_files/doc/source/faq.rst:264]: extensions; _> extensions from only the top level directory;

**Review by adswafford — CHANGES_REQUESTED** [review]
Couple of minor changes, but otherwise looks good. Thanks for updating this so quickly!

**Comment by antgonza** [comment]
No problem, thought it would be worth adding these changes (as they are documentation) for the next deployment. Thus, if the changes look good to you and you can merge them, that will be great!

**Pull request merged**